src/test/java/tdl/datapoint/infra_events/RecordingsDatapointAcceptanceTest.java

```diff
@@ -103,37 +100,40 @@ public void tearDown() throws Exception {
     }
 
     @Test
-    public void start_video_player() throws Exception {
-        // Given - The participant produces Video files while solving a challenge
+    public void start_recorder() throws Exception {
+        // Given - The participant produces Video or Source files while solving a challenge
         String challengeId = generateId();
         String participantId = generateId();
-        String s3destination = String.format("%s/%s/file.mp4", challengeId, participantId);
 
         // When - Upload event happens
-        S3Event s3Event1 = localS3Bucket.putObject(video1.asFile(), s3destination);
-        videoRecorderAlertHandler.handleRequest(
+        S3Event s3Event1 = createS3PutEventFrom(RECORDER_STARTED_EVENT1);
+        eventsAlertHandler.handleRequest(
                 convertToMap(wrapAsSNSEvent(s3Event1)),
                 NO_CONTEXT);
 
-        // Then - Repo is created with the contents of the SRCS file
+        // Then - Repo is created with the contents of the Video or SRCS file
         waitForQueueToReceiveEvents();
-        VideoRecorderStartedEvent queueEvent1 = videoRecorderStartedEvents.pop();
-        String repoUrl1 = queueEvent1.getVideoFileLink();
-        assertThat(repoUrl1, allOf(startsWith("file:///"),
-                containsString(challengeId),
+        RecorderStartedEvent queueEvent1 = recorderStartedEvents.pop();
+        String eventString1 = queueEvent1.toString();  // eventually might be a idea to verify the event sent getEventAsJsonString();
+        assertThat(eventString1, allOf(containsString(challengeId),
                 endsWith(participantId)));
 
         // When - Another upload event happens
-        S3Event s3Event2 = localS3Bucket.putObject(video2.asFile(), s3destination);
```

Comment:
No need for a second event, it is not proving anything.

Reply:
Ok will remove redundant check